### PR TITLE
feat(tier4_perception_launch): add radar fusion launch

### PR DESCRIPTION
**Pull request by scepter914**
## Description

Fixed reviewed points by <https://github.com/autowarefoundation/autoware.universe/pull/1263> <https://github.com/autowarefoundation/autoware.universe/pull/1285>.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 1]
```diff
@@ -296,7 +296,10 @@ def create_time_series_outlier_filter_components(input_topic, output_topic):
                 plugin="occupancy_grid_map_outlier_filter::OccupancyGridMapOutlierFilterComponent",
                 name="occupancy_grid_map_outlier_filter",
                 remappings=[
-                    ("~/input/occupancy_grid_map", "/perception/occupancy_grid_map/map"),
+                    (
+                        "~/input/occupancy_grid_map",
+                        "/perception/occupancy_grid_map/map",
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py:301]: I think your formatter change automatically. need to change? :pray:

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py [hunk 2]
```diff
@@ -337,7 +340,11 @@ def create_single_frame_outlier_filter_components(input_topic, output_topic):
                             ]
                         ),
                         "elevation_map_directory": PathJoinSubstitution(
-                            [FindPackageShare("elevation_map_loader"), "data", "elevation_maps"]
+                            [
+                                FindPackageShare("elevation_map_loader"),
+                                "data",
+                                "elevation_maps",
+                            ]
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py:347]: https://github.com/autowarefoundation/autoware.universe/pull/1294/files#r916703189
same comment

**Comment by yukkysaito** [comment]
@scepter914 Almost OK. Thank you :pray:

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 1 above)
Comment (scepter914) [reply launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py]: fix

File: launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py (hunk 2 above)
Comment (scepter914) [reply launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py]: fixed

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**